### PR TITLE
Fix typos and code brackets in coding guidelines

### PR DESCRIPTION
**Pull request by BaerMitUmlaut**
**When merged this pull request will:**
- Fix typos and code brackets in coding guidelines

### CODE REVIEW

**Comment by jonpas** [comment]
Confirmed, typo == bug!

**Comment by BaerMitUmlaut** [comment]
The "bug fix" label was more for the broken code boxes :wink:

**Comment by jonpas** [comment]
> more

So not solely. :laughing:

**Comment by jonpas** [comment]
L48 should probably be wrapped in code as well.

File: wiki/development/coding-guidelines.md [hunk 1]
```diff
@@ -298,7 +302,7 @@ Comments within the code should be used when they are describing a complex and c
 
 Good:
 
-```js
+```sqf
```
Comment (bux578) [inline wiki/development/coding-guidelines.md]: doesn't work

Comment (BaerMitUmlaut) [inline wiki/development/coding-guidelines.md]: I assumed it would because it was used somewhere else in that file. Will change.

**Pull request merged**